integration_tests/features/WalletRoutingMechanism.feature

```diff
@@ -1,43 +1,60 @@
-    Then all wallets detect all transactions are at least Completed
-    Then all wallets detect all transactions are at least Broadcast
+        When I wait 1 seconds
+        When I wait for wallet WALLET_A to have at least 100000000 uT
+        #When I print the world
+        And I multi-send 1000000 uT from wallet WALLET_A to all wallets at fee 100
+        Then all wallets detect all transactions are at least Pending
+        Then all wallets detect all transactions are at least Completed
+        Then all wallets detect all transactions are at least Broadcast
         # TODO: This wait is needed to stop next merge mining task from continuing
-    When I wait 1 seconds
-    And mining node MINER mines 1 blocks
-    Then all nodes are at height 21
-    Then all wallets detect all transactions as Mined_Unconfirmed
+        When I wait 1 seconds
+        And mining node MINER mines 1 blocks
+        Then all nodes are at height 21
+        Then all wallets detect all transactions as Mined_Unconfirmed
         # TODO: This wait is needed to stop next merge mining task from continuing
-    When I wait 1 seconds
-    And mining node MINER mines 11 blocks
-    Then all nodes are at height 32
-    Then all wallets detect all transactions as Mined_Confirmed
+        When I wait 1 seconds
+        And mining node MINER mines 11 blocks
+        Then all nodes are at height 32
+        Then all wallets detect all transactions as Mined_Confirmed
         # TODO: This wait is needed to stop base nodes from shutting down
-    When I wait 1 seconds
-    @critical
-    Examples:
-        | NumBaseNodes | NumWallets | Mechanism                |
-        |  5           |  5         | DirectAndStoreAndForward |
-        |  5           |  5         | DirectOnly               |
+        When I wait 1 seconds
+        @critical
+        Examples:
+            | NumBaseNodes | NumWallets | Mechanism                |
+            | 5            | 5          | DirectAndStoreAndForward |
+            | 5            | 5          | DirectOnly               |
 
-    @long-running
-    Examples:
-        | NumBaseNodes | NumWallets | Mechanism                |
-        |  5           |  5         | StoreAndForwardOnly      |
+        @long-running
+        Examples:
+            | NumBaseNodes | NumWallets | Mechanism           |
+            | 5            | 5          | StoreAndForwardOnly |
+
+    Scenario: Store and forward TX
+        Given I have a seed node SEED
+        And I have a base node BASE connected to seed SEED
+        And I have wallet SENDER connected to base node BASE
+        And I have wallet RECEIVER connected to base node BASE
+        And I stop wallet RECEIVER
+        And I have mining node MINE connected to base node BASE and wallet SENDER
+        And mining node MINE mines 5 blocks
+        Then I wait for wallet SENDER to have at least 1000000 uT
+        And I send 1000000 uT from wallet SENDER to wallet RECEIVER at fee 100
```

Comment:
I am working on a very similar test and ran into an issue when I spotted your WIP PR.

So firstly I found that this line would fail quite often because the way the `send_tari` helper function is written it uses the gRPC client to get the Receiver's pubkey but if you stopped the receiver it can't do that.

Reply:
I've changed the send_tari function (forgot to push, as I was still working on it). Now it's working, but it's the longest test we have :).